com.unity.visualeffectgraph/Shaders/ParticleHexahedron/Pass.template

```diff
@@ -28,8 +28,8 @@ VFX_VARYING_PS_INPUTS vert(uint id : SV_VertexID, vs_input i)
 	
 	float3 offsets = (float3)0;
 	offsets.x = float(id & 1);
-	offsets.y = float((id & 2) >> 1);
-	offsets.z = float((id & 6) >> 2);
+	offsets.y = (id & 2) * 0.5f;
+	offsets.z = (id & 4) * 0.25f;
```

Comment:
Why is it (id & 4) and before it was (id & 6)? I assume before it was a mistake, right?

Reply:
Yep most probably a typo but as it was shifted by 2 in previous code, &6 was equivalent to &4. Of course it wasnt equivalent anymore with new code.